dashboard/app/mailers/peer_review_mailer.rb

```diff
@@ -0,0 +1,13 @@
+class PeerReviewMailer < ActionMailer::Base
+  default from: 'Code.org <teacher@code.org>'
+
+  def review_completed_receipt(peer_review)
+    @peer_review = peer_review
+    @submission_url = CDO.studio_url @peer_review.submission_path, CDO.default_scheme
+
+    mail(
+      to: @peer_review.submitter.email,
+      subject: "Your submission for \"#{@peer_review.level.name}\" has been reviewed by a peer"
```

Comment:
Maybe say "has been reviewed." Some feedback will be from instructors, not peers